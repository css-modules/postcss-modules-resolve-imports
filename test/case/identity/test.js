'use strict';

const setup = require('../../setup');

test('identity', () => {
  const {resulting} = setup()(__dirname);

  expect(resulting).toMatchSnapshot();
});
