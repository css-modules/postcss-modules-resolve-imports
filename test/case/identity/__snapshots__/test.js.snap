// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`identity 1`] = `
".selector
{
  /* test for setupCase without plugins */
  rule: value;
}
"
`;
